### PR TITLE
Add changelog for v1.13.3

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,28 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.13.3](https://github.com/go-gitea/gitea/releases/tag/v1.13.3) - 2021-03-04
+
+* BREAKING
+  * Turn default hash password algorightm back to pbkdf2 from argon2 until we found a better one (#14673) (#14675)
+* BUGFIXES
+  * Print usefull error if SQLite is used in settings but not supported (#14476) (#14874)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Print useful error if SQLite is used in settings but not supported (#14476) (#14874)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,28 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix display since time round (#14226) (#14873)
+  * When Deleting Repository only explicitly close PRs whose base is not this repository (#14823) (#14842)
+  * Set HCaptchaSiteKey on Link Account pages (#14834) (#14839)
+  * Fix a couple of CommentAsPatch issues.  (#14804) (#14820)
+  * Disable broken OAuth2 providers at startup (#14802) (#14811)
+  * Backport: Repo Transfer permission checks (#14792) (#14794)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Repo Transfer permission checks (#14792) (#14794)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,28 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix double alert in oauth2 application edit view (#14764) (#14768)
+  * Fix broken spans in diffs (#14678) (#14683)
+  * Prevent race in PersistableChannelUniqueQueue.Has (#14651) (#14676)
+  * HasPreviousCommit causes recursive load of commits unnecessarily (#14598) (#14649)
+  * Do not assume all 40 char strings are SHA1s (#14624) (#14648)
+  * Allow org labels to be set with issue templates (#14593) (#14647)
+  * Accept multiple SSH keys in single LDAP SSHPublicKey attribute (#13989) (#14607)
+  * Fix bug about ListOptions and stars/watchers pagnation (#14556) (#14573)
+  * Fix GPG key deletion during account deletion (#14561) (#14569)
```
Comment (zeripath) [inline CHANGELOG.md:28]:
```suggestion
  * Fix GPG key deletion during account deletion (#14561) (#14569)
  * Fix paging of file commit logs (#14831) (#14879)
```

Once #14879 is merged

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,29 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.13.3](https://github.com/go-gitea/gitea/releases/tag/v1.13.3) - 2021-03-04
+
+* BREAKING
+  * Turn default hash password algorightm back to pbkdf2 from argon2 until we found a better one (#14673) (#14675)
+* BUGFIXES
+  * Print useful error if SQLite is used in settings but not supported (#14476) (#14874)
```
Comment (6543) [inline CHANGELOG.md:13]:
```suggestion
  * Fix paging of file commit logs (#14831) (#14879)
  * Print useful error if SQLite is used in settings but not supported (#14476) (#14874)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,29 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix double alert in oauth2 application edit view (#14764) (#14768)
+  * Fix broken spans in diffs (#14678) (#14683)
+  * Prevent race in PersistableChannelUniqueQueue.Has (#14651) (#14676)
+  * HasPreviousCommit causes recursive load of commits unnecessarily (#14598) (#14649)
+  * Do not assume all 40 char strings are SHA1s (#14624) (#14648)
+  * Allow org labels to be set with issue templates (#14593) (#14647)
+  * Accept multiple SSH keys in single LDAP SSHPublicKey attribute (#13989) (#14607)
+  * Fix bug about ListOptions and stars/watchers pagnation (#14556) (#14573)
+  * Fix GPG key deletion during account deletion (#14561) (#14569)
+  * Fix paging of file commit logs (#14831) (#14879)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

**Review by 6543 — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,29 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.13.3](https://github.com/go-gitea/gitea/releases/tag/v1.13.3) - 2021-03-04
+
+* BREAKING
+  * Turn default hash password algorightm back to pbkdf2 from argon2 until we found a better one (#14673) (#14675)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
  * Turn default hash password algorithm back to pbkdf2 from argon2 until we find a better one (#14673) (#14675)
```

**Review by techknowlogick — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

**Comment by 6543** [comment]
frontport -> #14883

**Pull request merged**